From 972669a570377fbff4a41cfa1564683b7a73647d Mon Sep 17 00:00:00 2001
From: Johnathan Corgan <johnathan@corganlabs.com>
Date: Tue, 31 May 2016 12:46:25 -0700
Subject: [PATCH 44/46] Revert "cmake: Fix warning related to GrSetupQt4 in
 modern CMake"

This reverts commit a62eea47e38313ebabe787d4b972b0bbb3af9ada.
---
 cmake/Modules/GrSetupQt4.cmake | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/cmake/Modules/GrSetupQt4.cmake b/cmake/Modules/GrSetupQt4.cmake
index cc85c39..312b415 100644
--- a/cmake/Modules/GrSetupQt4.cmake
+++ b/cmake/Modules/GrSetupQt4.cmake
@@ -67,7 +67,13 @@ set(__INCLUDED_GR_USEQT4_CMAKE TRUE)
 ################################################################
 
 ADD_DEFINITIONS(${QT_DEFINITIONS})
-SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS $<$<NOT:$<CONFIG:Debug>>:QT_NO_DEBUG>)
+SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS_DEBUG QT_DEBUG)
+SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS_RELEASE QT_NO_DEBUG)
+SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS_RELWITHDEBINFO QT_NO_DEBUG)
+SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS_MINSIZEREL QT_NO_DEBUG)
+IF(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
+  SET_PROPERTY(DIRECTORY APPEND PROPERTY COMPILE_DEFINITIONS QT_NO_DEBUG)
+ENDIF()
 
 SET(QT_INCLUDE_DIRS ${QT_INCLUDE_DIR})
 SET(QT_LIBRARIES "")
-- 
2.1.4

